src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -57,6 +57,9 @@ public class Drivetrain extends SubsystemBase {
   private double m_currentTranslationDir = 0.0;
   private double m_currentTranslationMag = 0.0;
 
+  // A percentage value (0-1) for the linear speed of the robot
+  public double m_maxSpeed = 0.0;
```

Comment:
Is there a reason this is public?

Do not make any instance or class variable public without a good reason. (Often variables do not need to be explicitly set or gotten – often that happens as a side effect of method calls)